packages/cli/src/utils/generate.ts

```diff
@@ -60,6 +60,31 @@ function filterAndCopyPackageJson(basePath: string) {
   })
 }
 
+// Temporary array of strict rules enabled so far.
+const TS_CONFIG_STRICT_RULES_ENABLED = ['noImplicitAny'] as const
+
+/**
+ * Modify TypeScript compilation settings (tsconfig.json) to disable specific strict
+ * type checking rules when files are moved to the .faststore folder. 
+ * When all strict rules are migrated. The idea is to only change the strict to false
+ */
+function disableTsConfigStrictRules(basePath: string) {
```

Comment:
To avoid problems when building stores, strict mode is disabled. It is only needed for people who maintain the `@faststore/core` project